classes/helper.php

```diff
@@ -306,6 +312,9 @@ public static function plain_text_search(string $search, string $table,
 
                     $count = 0;
                     foreach ($records as $record) {
+                        if ( ! empty($linkfunction)) {
+                            $linkstring = $linkfunction($record);
+                        }
```

Comment:
I think this and the regex version could be simplified as

`$linkstring = !empty($linkfunction) ? $linkfunction($record) : '';`

But not too concerned either way.

Reply:
I'll leave it as is.  I tend to avoid those ternary operators, unless I need to do it in an expression.